### PR TITLE
SFW-147: Updates to LICENSE, README & podspec

### PR DESCRIPTION
**Pull request by epeschard**
## Description

Ticket: [SFW-147](https://stanwood.atlassian.net/browse/SFW-147)

✅ Update licence in .podspec to MIT
✅ Update README licence to MIT
✅ Update LICENCE file to MIT
✅ Set s.source to HTTP, instead of ssh
✅ Update Usage(Installation guide)
✅ Replace the headers in all source files. and include copyright and MIT licence
✅ Replace the string in the Example launch screen to state copyright Stanwood GmbH.
✅ Check spelling of class and folder names.
✅ Cleanup the Podspec file and ensure that it has both a meaningful summary and description.
✅ Ensure the the Podfile in the example project has "platform :ios, 10".
✅ Version bump 
❌ push to master repo

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -76,4 +74,4 @@ Eugène Peschard, eugene.peschard@stanwood.io
 
 ## License
 
-StanwoodCore is a private library. See the [LICENSE](https://github.com/stanwood/Stanwood_Dialog_iOS/blob/master/LICENSE "Copyright © 2018 stanwood GmbH") file for more info.
+StanwoodCore is under MIT licence. See the [LICENSE](https://github.com/stanwood/Stanwood_Dialog_iOS/blob/master/LICENSE "Copyright © 2018 stanwood GmbH") file for more info.
```
Comment (talezion) [inline README.md]: @epeschard Change StanwoodCore to StanwoodDialog_iOS

File: StanwoodDialog.podspec [hunk 2]
```diff
@@ -1,5 +1,5 @@
 #
-# Be sure to run `pod lib lint Stanwood_Dialog_iOS.podspec' to ensure this is a
+# Be sure to run `pod lib lint StanwoodDialog.podspec' to ensure this is a
 # valid spec before submitting.
 #
 # Any lines starting with a # are optional, but their use is encouraged
```
Comment (talezion) [inline StanwoodDialog.podspec]: @epeschard Remove the podspec header

File: Stanwood_Dialog_iOS/Classes/RatingDialog.swift [hunk 3]
```diff
@@ -13,15 +13,17 @@ import UIKit
...
+public class RatingDialog: NSObject, RatingDialogPresenting {
     
     private var text1: String?
     private var text2: String?
     private var text3: String?
     private var text4: String?
     private var faceURL: URL?
     private var bannerURL: URL?
-    private var appStoreURL: URL?
+    private var appStoreURL: URL!
```
Comment (talezion) [inline Stanwood_Dialog_iOS/Classes/RatingDialog.swift]: @epeschard Why force unwrapping?

Comment (epeschard) [reply Stanwood_Dialog_iOS/Classes/RatingDialog.swift]: I was trying to remove the force unwrapping bellow now done properly with optional binding

File: Example/Stanwood_Dialog_iOS/AppDelegate.swift [hunk 4]
```diff
@@ -1,9 +1,9 @@
 //
 //  AppDelegate.swift
-//  Stanwood_Dialog_iOS
+//  StanwoodDialog_iOS
 //
 //  Created by epeschard on 01/03/2018.
-//  Copyright (c) 2018 epeschard. All rights reserved.
+//  Distributed under MIT licence.
```
Comment (talezion) [inline Example/Stanwood_Dialog_iOS/AppDelegate.swift]: @epeschard `Copyright (c) 2018 stanwood GmbH`

File: Example/Stanwood_Dialog_iOS/ViewController.swift [hunk 5]
```diff
@@ -1,9 +1,9 @@
 //
 //  ViewController.swift
-//  Stanwood_Dialog_iOS
+//  StanwoodDialog_iOS
 //
 //  Created by epeschard on 01/03/2018.
-//  Copyright (c) 2018 epeschard. All rights reserved.
+//  Distributed under MIT licence.
```
Comment (talezion) [inline Example/Stanwood_Dialog_iOS/ViewController.swift]: @epeschard `Copyright (c) 2018 stanwood GmbH`

File: StanwoodDialog.podspec [hunk 6]
```diff
@@ -1,32 +1,17 @@
...
-# valid spec before submitting.
-#
-# Any lines starting with a # are optional, but their use is encouraged
-# To learn more about a Podspec see http://guides.cocoapods.org/syntax/podspec.html
-#
-
 Pod::Spec.new do |s|
   s.name             = 'StanwoodDialog'
-  s.version          = '0.7.1'
+  s.version          = '0.7.2'
```
Comment (talezion) [inline StanwoodDialog.podspec]: @epeschard pump to `1.0`

File: Stanwood_Dialog_iOS/Classes/RatingDialog.swift [hunk 7]
```diff
@@ -1,9 +1,30 @@
 //
 //  RatingDialog.swift
-//  Pods-Stanwood_Dialog_iOS_Example
+//  Pods-StanwoodDialog_iOS_Example
 //
-//  Created by Eugène Peschard on 05/01/2018.
+//  Created by epeschard on 05/01/2018.
```
Comment (talezion) [inline Stanwood_Dialog_iOS/Classes/RatingDialog.swift]: @epeschard `Copyright (c) 2018 stanwood GmbH`

File: Stanwood_Dialog_iOS/Classes/RatingDialogTracking.swift [hunk 8]
```diff
@@ -4,6 +4,27 @@
 //
 //  Created by Ronan on 10/01/2018.
```
Comment (talezion) [inline Stanwood_Dialog_iOS/Classes/RatingDialogTracking.swift]: @epeschard We need to remove and add the file name .swift

File: Stanwood_Dialog_iOS/Classes/RatingDialogView.swift [hunk 9]
```diff
@@ -2,8 +2,29 @@
 //  RatingDialogView.swift
 //  StanwoodDialog
 //
-//  Created by Eugène Peschard on 08/01/2018.
+//  Created by epeschard on 08/01/2018.
```
Comment (talezion) [inline Stanwood_Dialog_iOS/Classes/RatingDialogView.swift]: @epeschard Please remove

**Pull request merged**